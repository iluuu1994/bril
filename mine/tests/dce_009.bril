@main {
  a: int = const 1;
  a: int = add a a;
  a: int = const 2;
  print a;
}
