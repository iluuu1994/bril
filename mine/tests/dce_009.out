@main {
  a: int = const 2;
  a: int = id a;
  print a;
}
